clusters/app.ci/cert-manager/rhcos-redirector_ingress.yaml

```diff
@@ -3,6 +3,8 @@ kind: Ingress
 metadata:
   annotations:
     cert-manager.io/cluster-issuer: cert-issuer
+  labels:
+    type: tlsv11
```

Comment:
This label will be copied over to all of the generated routes from this ingress.